Description: Upstream changes introduced in version 0.2.22-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 partclone (0.2.22-1) unstable; urgency=low
 .
   * Initial release (Closes: #623452)
   * removed the upstream's debian subdirectory, and taken some of its contents
     as an input for the new debian subdirectory generated by dh_make
   * refreshed the list of build-dependencies which had been defined by
     upstream developers: many filesystem libraries have been merged into
     libc6, and are part of the current kernels
 .
 The person named in the Author field signed this changelog entry.
Author: Georges Khaznadar <georgesk@ofset.org>
Bug-Debian: http://bugs.debian.org/623452

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- partclone-0.2.22.orig/configure.ac
+++ partclone-0.2.22/configure.ac
@@ -22,16 +22,16 @@ AC_ARG_ENABLE(all,
 )
 AM_CONDITIONAL(ENABLE_ALL, test "$enable_all" = yes)
 if test "$enable_all" = "yes"; then
-enable_xfs="yes"
+#enable_xfs="yes"
 enable_extfs="yes"
-enable_reiserfs="yes"
-enable_reiser4="yes"
+#enable_reiserfs="yes"
+#enable_reiser4="yes"
 enable_hfsp="yes"
 enable_fat="yes"
 enable_ntfs="yes"
-enable_ufs="yes"
-enable_vmfs="yes"
-enable_jfs="yes"
+#enable_ufs="yes"
+#enable_vmfs="yes"
+#enable_jfs="yes"
 enable_btrfs="yes"
 fi
 
@@ -56,7 +56,7 @@ AC_CHECK_HEADERS([ext2fs/ext2fs.h], ,
 AC_CHECK_LIB([ext2fs], [ext2fs_initialize], true,
     AC_MSG_ERROR([*** EXT2/3 library (libext2fs) not found]))
 AC_MSG_CHECKING(version of libextfs)
-extfs_version=`gcc -static  $srcdir/src/deplib_version.c -o $srcdir/get_lib_version -lext2fs -DEXTFS`
+extfs_version=`gcc  $srcdir/src/deplib_version.c -o $srcdir/get_lib_version -lext2fs -DEXTFS`
 extfs_version=`$srcdir/get_lib_version extfs | cut -d'.' -f1`
 extfs_version_major=`$srcdir/get_lib_version extfs | cut -d'.' -f2`
 extfs_version_minor=`$srcdir/get_lib_version extfs | cut -d'.' -f3`
@@ -131,7 +131,7 @@ AC_CHECK_LIB([reiserfs], [reiserfs_fs_op
 AC_CHECK_LIB([dal], [file_dal_open], true,
     AC_MSG_ERROR([*** Reiserfs depend library (libdal) not found]))
 AC_MSG_CHECKING(version of libreiserfs)
-reiserfs_version=`gcc -static  $srcdir/src/deplib_version.c -o $srcdir/get_lib_version -lreiserfs -DREISERFS`
+reiserfs_version=`gcc  $srcdir/src/deplib_version.c -o $srcdir/get_lib_version -lreiserfs -DREISERFS`
 reiserfs_version=`$srcdir/get_lib_version reiserfs | cut -d'.' -f1`
 reiserfs_version_major=`$srcdir/get_lib_version reiserfs | cut -d'.' -f2`
 reiserfs_version_minor=`$srcdir/get_lib_version reiserfs | cut -d'.' -f3`
@@ -184,7 +184,7 @@ AC_CHECK_HEADERS([reiser4/libreiser4.h],
 AC_CHECK_LIB([aal], [aal_device_open], true,
     AC_MSG_ERROR([*** Reiser4 depend library (libaal) not found]))
 AC_MSG_CHECKING(version of libreiser4)
-reiser4_version=`gcc -static  $srcdir/src/deplib_version.c -o $srcdir/get_lib_version -lreiser4 -laal -DREISER4`
+reiser4_version=`gcc  $srcdir/src/deplib_version.c -o $srcdir/get_lib_version -lreiser4 -laal -DREISER4`
 reiser4_version=`$srcdir/get_lib_version reiser4 | cut -d'.' -f1`
 reiser4_version_major=`$srcdir/get_lib_version reiser4 | cut -d'.' -f2`
 reiser4_version_minor=`$srcdir/get_lib_version reiser4 | cut -d'.' -f3`
@@ -259,7 +259,7 @@ AC_CHECK_HEADERS([ntfs/version.h], ,
 AC_CHECK_LIB([ntfs], [ntfs_mount], true,
    AC_MSG_ERROR([*** ntfs depend library (libntfs) not found]))
 AC_MSG_CHECKING(version of libntfs)
-ntfs_version=`gcc -static  $srcdir/src/deplib_version.c -o $srcdir/get_lib_version -lntfs -DNTFS`
+ntfs_version=`gcc  $srcdir/src/deplib_version.c -o $srcdir/get_lib_version -lntfs -DNTFS`
 ntfs_version=`$srcdir/get_lib_version ntfs | cut -d':' -f1`
 ntfs_version_major=`$srcdir/get_lib_version ntfs | cut -d':' -f2`
 ntfs_version_minor=`$srcdir/get_lib_version ntfs | cut -d':' -f3`
